poetry.lock

```diff
@@ -9,7 +9,7 @@ version = "1.4.3"
 [[package]]
```

Comment:
the lock file would work just fine. Do you have any idea why the previous lock file broke?

Reply:
I really have no idea. I seem to have recurring issues with lock files and poetry in general. This time the error was

`[KeyAlreadyPresent]
Key "extras" already exists.`

Googling it I got the impression this was another case of pip and poetry not liking each other.

It's not really an issue since these problems are always fixed by just deleting the lock file and regenerating it. My only concern is that it might break something for others. We could really use some CI tools at some points, like we discussed, on the whole framework level, just to check all the modules play nice with each other.

Reply:
The CI would be a nice thing to have. Let's wait until we are ready to release version 1